packages/theming/theme-types/src/Color.types.ts

```diff
@@ -533,6 +535,7 @@ export interface AliasColorTokens {
   brandBackground: ColorValue;
   brandBackgroundHover: ColorValue;
   brandBackgroundPressed: ColorValue;
+  brandBackgroundDisabled: ColorValue;
```

Comment:
Is this color different from `neutralBackgroundDisabled`? For other platforms the disabled color has been the same across all backgrounds, which is why there's only one `neutralBackgroundDisabled` token